src/pages/groupComparison/comparisonGroupManager/GroupCheckbox.tsx

```diff
@@ -98,6 +98,35 @@ export default class GroupCheckbox extends React.Component<IGroupCheckboxProps,
         );
     }
 
+    @computed get editingNameUI() {
+        return (
+            <DefaultTooltip
+                visible={!!(this.editSubmitError && this.editSubmitError.message)}
+                overlay={
+                    <div>
+                        <i
+                            className="fa fa-md fa-exclamation-triangle"
+                            style={{
+                                color: "#BB1700",
+                                marginRight: 5
+                            }}
+                        />
+                        <span>
+                                    {this.editSubmitError && this.editSubmitError.message}
+                                </span>
+                    </div>
+                }
+            >
+                <input
+                    type="text"
+                    value={this.nameInput}
+                    onChange={this.handleNameInputChange}
+                    style={{width:174}}
+                />
+            </DefaultTooltip>
+        );
+    }
```

Comment:
I don't see any content changes for this block. Let me know if it's not the case.

Reply:
thats right